### PR TITLE
Html CSS PR

### PR DESCRIPTION
**Pull request by IsmailChangezi**
Added markup With HTML
Added Style with CSS
Corrected Linter errors
Corrected Stylelint errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        </p>
+        <div class="icons">
+          <i class="fa-brands fa-github"></i>
+          <i class="fa-brands fa-linkedin-in"></i>
+          <i class="fa-brands fa-angellist"></i>
+          <i class="fa-brands fa-facebook"></i>
+          <i class="fa-brands fa-twitter"></i>
+        </div>
+      </section>
+    </header>
```
Comment (ghost) [inline index.html:39]: - [ ] The toolbar and the headline should be two different sections. Would you please separate them
   - Wrap your toolbar with the `<header>` tag
   - Wrap your headline with the `<section>` tag
- [ ] This is important for [semantic and accessibility reasons](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <h5>Ismail Changezi</h5>
```
Comment (ghost) [inline index.html:20]: - [ ] Would you please wrap your logo with an `<a>`  tag  to look like `<a>Ismail Changezi</a>` . Apart from being semantically correct, the logo will be redirecting you to the home page in the future.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <i class="fa-solid fa-bars"></i>
```
Comment (ghost) [inline index.html:21]: - [ ] Please wrap your hamburger button with a `<nav>`  tag for semantic and accessibility reasons

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+      </nav>
+      <section>
+        <h4>Hey There.<br />I'm Ismail</h4>
```
Comment (ghost) [inline index.html:24]: - [ ] Would you please wrap your main title with a `<h1>` tag
   - If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        <h6>I'm a Software Developer</h6>
```
Comment (ghost) [inline index.html:25]: - [ ] Would you please use a `<p>` tag for semantic reasons.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="icons">
+          <i class="fa-brands fa-github"></i>
+          <i class="fa-brands fa-linkedin-in"></i>
+          <i class="fa-brands fa-angellist"></i>
+          <i class="fa-brands fa-facebook"></i>
+          <i class="fa-brands fa-twitter"></i>
+        </div>
```
Comment (ghost) [inline index.html:37]: - [ ] Would you please  `<ul>` tag to group your social links icons and `<li>` tag  for each icon for accessibility reasons
- [ ] would you please wrap each icon with an `<a>` tag.  Apart from being semantically correct, the `<a>` tag will let you connect your social networks with the icons, using href.

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost there

## Highlights

- Have used the right GitHub flow ✅
- Linter checks passed ✅

## Required Changes ♻️

- [REQUIRED] Your Readme file is incomplete. Would you please use  the Readme template provided by Microverse and customize it

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by IsmailChangezi** [comment]
Updated the README.md file.
I will work on the optional requirements once this gets approved.

File: index.html (hunk 2 above)
Comment (leonardodiasb) [inline index.html:20]: - [ ] Would you please wrap your logo with an `<a>` tag to look like `<a>Ismail Changezi</a>`. Apart from being semantically correct, the logo will be redirecting you to the home page in the future.

File: index.html (hunk 4 above)
Comment (leonardodiasb) [inline index.html:24]: - [ ] Kindly wrap your main title with a `<h1>` tag
If you want that google (and other search engines) knows about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

File: index.html (hunk 5 above)
Comment (leonardodiasb) [inline index.html:25]: - [ ] Kindly use a `<p>` tag for semantic reasons.

File: index.html (hunk 6 above)
Comment (leonardodiasb) [inline index.html:37]: - [ ]  Kindly use `<ul>` tag to group your social links icons and `<li>` tag for each icon for accessibility reasons.
- [ ]  Please wrap each icon with an `<a>` tag. Apart from being semantically correct, the <a> tag will let you connect your social networks with the icons, using href.

File: index.html (hunk 1 above)
Comment (leonardodiasb) [inline index.html:39]: - [ ] The toolbar and the headline should be two different sections. Would you please separate them
   - Wrap your toolbar with the `<header>` tag
   - Wrap your headline with the `<section>` tag

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @IsmailChangezi, 

I see that you did not implement any changes asked by the previous Code Reviewer. If you are struggling with something you can reach me or other Code Reviewers on Slack, or if you disagree with the review you can submit a ticket to Student Success. Unfortunately, I cannot approve this PR until you implement those changes.

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Your Readme file is incomplete. Would you please use the Readme template provided by Microverse and customize it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by IsmailChangezi** [comment]
Updated README.md files in both branches

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @IsmailChangezi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## ❌ Invalid Code Review Request

You have submitted a project, that the linters are not running. I think this might be due to the merge conflicts. Kindly fix your merge conflicts and check if the linters will run. 

Let me know if you couldn't fix the merge conflicts so that we will fix them together 😄 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request closed without merging**